cmd/soroban-rpc/lib/preflight/src/lib.rs

```diff
@@ -236,8 +236,7 @@ fn host_events_to_c(events: Events) -> Result<*mut *mut libc::c_char, Box<dyn er
         let maybe_contract_event = match &e.event {
             Event::Contract(e) => Some(e),
             Event::StructuredDebug(e) => Some(e),
-            // TODO: should we (somehow) map the debug events to XDR?
-            //       I (fons) am not even sure that's possible
+            // Debug events can't be translated to diagnostic events
             Event::Debug(_) => None,
```

Comment:
@sisuresh Can you confirm this is correct?

Reply:
Yeah this is right. We have an issue open to consolidate Debug events into StructuredDebug https://github.com/stellar/rs-soroban-env/issues/731.

Reply:
Thanks!